php/commands/src/CLI_Alias_Command.php

```diff
@@ -0,0 +1,335 @@
+	 *     $ wp cli alias add @prod login@host /path/to/wordpress/install/
+	 *     Success: Added '@prod' alias.
+	 *
+	 *     # Add alias to project config.
+	 *     $ wp cli alias add @prod login@host /path/to/wordpress/install/ --config=project
+	 *     Success: Added '@prod' alias.
+	 *
+	 * @subcommand add
+	 */
+	public function add( $args, $assoc_args ) {
+
+		$config = ( ! empty( $assoc_args['config'] ) ? $assoc_args['config'] : 'global' );
+
+		list( $config_path, $aliases ) = $this->_get_aliases_data( $config, '' );
+
+		$this->_validate_config_file( $config_path );
+
+		list( $alias, $login, $wp_path ) = $args;
+
+		if ( ! isset( $aliases[ $alias ] ) ) {
+			$aliases[ $alias ]['ssh'] = $login . ':' . $wp_path;
+			$yaml_data                = Spyc::YAMLDump( $aliases );
+
+			if ( file_put_contents( $config_path, $yaml_data ) ) {
+				WP_CLI::success( sprintf( 'Added \'%s\' alias.', $alias ) );
+			}
+		} else {
+			WP_CLI::error( sprintf( 'Key \'%s\' exists already!', $alias ) );
+		}
+	}
+
+	/**
+	 * Deletes an alias.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Key for the alias.
+	 *
+	 * [--config=<config>]
+	 * : Config file to be considered for operations.
+	 * ---
+	 * options:
+	 *   - global
+	 *   - project
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Delete alias.
+	 *     $ wp cli alias delete @prod
+	 *     Success: Deleted '@prod' alias.
+	 *
+	 *     # Delete project alias.
+	 *     $ wp cli alias delete @prod --config=project
+	 *     Success: Deleted '@prod' alias.
+	 *
+	 * @subcommand delete
+	 */
+	public function delete( $args, $assoc_args ) {
+
+		list( $alias ) = $args;
+
+		$config = ( ! empty( $assoc_args['config'] ) ? $assoc_args['config'] : '' );
+
+		list( $config_path, $aliases ) = $this->_get_aliases_data( $config, $alias );
+
+		$this->_validate_config_file( $config_path );
+
+		if ( ! empty( $aliases[ $alias ] ) ) {
+			unset( $aliases[ $alias ] );
+			$yaml_data = Spyc::YAMLDump( $aliases );
+
+			if ( file_put_contents( $config_path, $yaml_data ) ) {
+				WP_CLI::success( sprintf( 'Deleted \'%s\' alias.', $alias ) );
+			}
+		} else {
+			WP_CLI::error( sprintf( 'No alias found with key \'%s\'.', $alias ) );
+		}
+	}
+
+	/**
+	 * Updates an alias.
+	 *
+	 * ## OPTIONS
+	 *
+	 * <key>
+	 * : Key for the alias.
+	 *
+	 * <login>
+	 * : SSH Login.
+	 *
+	 * <wp_path>
+	 * : WordPress install path.
+	 *
+	 * [--config=<config>]
+	 * : Config file to be considered for operations.
+	 * ---
+	 * options:
+	 *   - global
+	 *   - project
+	 * ---
+	 *
+	 * ## EXAMPLES
+	 *
+	 *     # Update alias.
+	 *     $ wp cli alias update @prod newlogin@host /updated/wordpress/path/
+	 *     Success: Updated 'prod' alias.
+	 *
+	 *     # Update project alias.
+	 *     $ wp cli alias update @prod newlogin@host /updated/wordpress/path/ --config=project
+	 *     Success: Updated 'prod' alias.
+	 *
+	 * @subcommand update
+	 */
+	public function update( $args, $assoc_args ) {
+
+		list( $alias, $login, $wp_path ) = $args;
```

Comment:
This would need to be adapted in a similar way than the `add` command to allow for updating all of the possible values.